ivadomed/config/config.json

```diff
@@ -3,6 +3,9 @@
     "gpu": 7,
     "log_directory": "spineGeneric",
     "debugging": false,
+    "object_detection_params": {
+        "object_detection_path": null
```

Comment:
in another PR (https://github.com/neuropoly/ivado-medical-imaging/pull/274) we discussed the use of "null" in json files, vs. empty string. We should probably settle on a convention.
@Drulex @kousu do you have any suggestions about what to use / not to use?

Reply:
Simple note: in #274, empty string have been used instead of `null`.